### ISSUE TITLE
Accessibility violation on `<a class="chosen-single chosen-default">`.

### ISSUE THREAD

**Issue by tlo450** [noli42/chosen#9]
Hey all,

I'm using this version of the chosen library in a Drupal 10 project, and I just want to report an accessibility violation on pages where it's used:

"This A lacks an href or a tabindex set to 0 or a positive value."

It looks like something similar to the issue over on the JJJ fork at https://github.com/JJJ/chosen/issues/52. However I think this may be something different as I still see the violation I mentioned above even if I had `role="button"` to that anchor element.

**Comment by noli42** [comment]
Thanks for the report!

Can you reproduce the issue at https://noli42.github.io/chosen/docs/ ?

Is there an easy way to test this?

**Comment by tlo450** [comment]
I'm currently using a Chrome extension 'Access Assistant' to scan for accessibility violations, and if I scan https://noli42.github.io/chosen/docs/  I am seeing the same error:

<img width="755" height="281" alt="Image" src="https://github.com/user-attachments/assets/c1832016-b4a9-4cf8-8e78-3c378ca9608c" />

I'm also seeing the same violation when scanning with other accessibility extensions.

**Comment by noli42** [comment]
Can you try again on https://noli42.github.io/chosen/docs/ after ctrl + f5

**Comment by tlo450** [comment]
Thanks for looking at this so fast!

The violation is now gone if I run a scan on that page 👍 

There are some other violations I'm seeing on that demo page, but I'm not seeing any of them in Drupal on the pages where I'm using chosen, so I can't speak to those.

**Comment by noli42** [comment]
Can you check again?
https://noli42.github.io/chosen/docs/

I have done some improvements on the library and the docs, and I got 96 on the accessibility score now in devtools lighthouse test.

**Comment by tlo450** [comment]
Oh wow, most of the violations have gone away, nice 👍  If you are curious, the only ones I'm seeing now I don't even think really are valid violations:

<img width="716" height="284" alt="Image" src="https://github.com/user-attachments/assets/579be299-505c-4fda-bc24-865a14cff624" />

Apparently for 'JJJ', my tool is saying "This A element has a suspicious (i.e. lacks purpose or is >150 characters) calculated accessible name value of: jjj".

The other ones are: This A element has an accessible name of 'harvest' that is the same as another A element but doesn't go to the same HREF.

**Comment by noli42** [comment]
Nice!

Probably we can ignore those links then.

I will make a new release soon.
